src/scalper.ts

```diff
@@ -82,22 +85,38 @@ export class Scalper {
       this.perpMarketConfig.quoteDecimals
     );
 
+    // Open Mango Websocket
+    const fillFeed = new WebSocket(FILLS_URL!);
+    fillFeed.onopen = function(e) {
+      console.log('Connected to Mango Websocket', new Date().toUTCString())
+    };
+    //TODO error handling
```

Comment:
What error handling are you thinking of? Is it safe to just do a cancelStaleOrders and read the state, or do we need to do some kind of cleanup when we get an error?

Reply:
I have observed occasionally issues where it seems the websocket missed a fill, couldnt identify whether we got disconnected or if it was an issue from mango's side. 

cancelStaleOrders and rerunning is a good safety for anything unexpected occurring anywhere in the code

Reply:
Roger this was prior to your commit separately on the top level error handling